FordLabs/PeopleMover
# 133 add person tags

## Issue
Resolves 133

## What was done
- [x] Added people tags as a concept to the app
- [x] Added field for creating / adding people tags to a person
 
## How to test
1. Create / edit a person
2. Observe the "Person Tags" section
3. Create a tag
4. Observe the tag being assigned to the person
5. Go to a different person
6. Open Person Tags dropdown
7. Select the tag you previously created
8. Observe the tag exists
9. Go back to the first person
10. Observe the tag is still assigned
11. Remove the tag
12. Observe the tag is removed
